src/pages/GalleryPage.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from "react";
+import Card from "../components/Card";
+import { useSelector, useDispatch } from "react-redux";
+import useGetImages from "hooks/useGetImages";
+import Masonry, { ResponsiveMasonry } from "react-responsive-masonry";
+import { setLoadmore } from "redux/actions";
+
+export default function GalleryPage() {
+  const { filter } = useSelector((state) => state);
+  const { images } = useGetImages();
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(setLoadmore(true));
+  }, []);
```

Comment:
I'm actually getting a linting error from this line:
```
src/pages/GalleryPage.js
  Line 14:6: React Hook useEffect has a missing dependency: 'dispatch'. Either include it or remove the dependency array 
```

Do you know why eslint would warn us about this? How might you fix this?